### ISSUE TITLE
Integration with mocha

### ISSUE THREAD

**Issue by sidorares** [sidorares/node-vim-debugger#28]
I often use cli debugger with mocha via `mocha debug test/foo/bar`. It would be great to have `mocha vimdebug test/foo/bar` (if possible without modification to mocha, via mocha.opts + --require )

**Comment by DelvarWorld** [comment]
@sidorares have you gotten this to work with vanilla mocha? that's what i'm looking at now

**Comment by sidorares** [comment]
@DelvarWorld no, I haven't looked at this yet

**Comment by sidorares** [comment]
@DelvarWorld afaik you can start mocha tests with `--debug-brk`, that way workflow should be identical as documented in the readme

**Comment by Seikho** [comment]
You can also debug the `mocha` unit tests by running by debugging `node_modules/mocha/bin/_mocha` from your root project folder. This assumes you have mocha installed as a (dev-)dependency in your project (as it should be).

**Comment by sidorares** [comment]
yes, I'm just thinking about best way on how to integrate this for mocha users which would require wery little configuration ( line in mocha.opts or something similar )

**Comment by Seikho** [comment]
Instructions in the readme should be sufficient, in my opinion.
Create a `debug.js` in the root project require that contains the code:
```javascript
require('node_modules/mocha/bin_mocha')
```

Then use `node-vim-inspector debug.js` to get started.
